developers/ocean.py/README.md

```diff
@@ -10,13 +10,13 @@ Well, brace yourselves for some exhilarating news! Introducing ocean.py, a Pytho
 
```

Comment:
Replace paragraph "Well, brace..." with:

"This is where ocean.py can help. It is a Python library for data scientists to privately & securely publish, exchange, and consume data."

Why: the previous version didn't emphasize that ocean.py is for data scientists; was too long; had too many distracting emojis; was too enthusiastic; and didn't properly summarize what ocean.py actually _does_.